service/hook/common/common.go

```diff
@@ -43,6 +43,8 @@ type TransformResponseModel struct {
 type TransformResponseInputModel struct {
 	// Errors include the errors if the build could not trigger
 	Errors []string
+	SkipResponses        []bitriseapi.SkipAPIResponseModel
```

Comment:
to conform to the naming of the other related params rename this to `SkippedTriggerResponses`